docs/release-guide.md

```diff
@@ -31,14 +31,17 @@ For each release to be published, proceed as follows:
 * `cargo clean`
 * `git clean -fd`
 * `export RELEASE_VER=x.y.z`
+* Depending on which version level this release increases, choose one of *major*, *minor*, or *patch* according to [semantic versioning][semver]
```

Comment:
If we need to bump major or minor, yes. That happens due to an API change/break anyway, so it can be done in the same PR. As I said, 0.2.1 flow was different because 0.2.0 never really happened.

Reply:
I don't think it was. I introduced consistency by manually bumping the version in 621eac2bc9d4fd2180924ea27e8a0ddfe2351dc9, so that `cargo release` was able to make the changes on its own.

Reply:
If I try to `cargo release` then it'll suggest a patch-level bump. In case I wanted to bump another level I'd have to specify that. What am I missing?

Reply:
See a real-world example at https://github.com/coreos/coreos-metadata/commit/01767dfefac63d59d77e71ee04ea1880854dda93#diff-80398c5faae3c069e4e6aa2ed11b28c0. The corresponding `cargo-release` run then was https://github.com/coreos/coreos-metadata/pull/115/commits.

Reply:
I'll wait this one out until the next release ;-)